system-test/test-e2e.ts

```diff
@@ -394,10 +396,12 @@ describe('@google-cloud/debug end-to-end behavior', () => {
             expressions: ['o'],
             log_message_format: 'o is: $0'
           });
+          // TODO: Determine if this type signature is correct.
         })
         .then((results: stackdriver.Breakpoint[]) => {
           // Check that the breakpoint was set, and then wait for the log to be
           // written to
+
```

Comment:
This looks like an unnecessary empty line.

Reply:
It looks like it's making the format more consistent with other sections...